appveyor.yml

```diff
@@ -3,17 +3,20 @@ environment:
     - nodejs_version: '8'
     - nodejs_version: '6'
     - nodejs_version: '4'
+
 install:
   - ps: Install-Product node $env:nodejs_version
   - set CI=true
   - npm install --global npm@latest
   - set PATH=%APPDATA%\npm;%PATH%
   - npm install
-matrix:
-  fast_finish: true
-build: off
-shallow_clone: true
+
+cache:
+  - node_modules
```

Comment:
why we cache `node_modules` BTW? we don't cache this folder on Travis

Reply:
In the hopes that it would result in faster build times. Seems like a bad idea in hindsight. I'll get rid of it in #273. Evidently npm doesn't like being told when to cache.

Reply:
yeah, and also #238 

but as a note, not to cache `node_modules` won't fix https://github.com/okonet/lint-staged/pull/273#issuecomment-328259546 :( it's a bug on npm v5.4 itself

Reply:
Ha ha.. With npm you can never be too sure. The build passed now 😆.

Reply:
LOL, yeah sometimes the problem comes from its cache, permission, node_modules, or even the npm itself 😆

Reply:
But on other projects which I never cache `node_modules`, the builds failed. Then set them to just use the bare default npm version fix the problems 🤔